shared/common-adapters/tabs.tsx

```diff
@@ -88,30 +116,46 @@ const styles = Styles.styleSheetCreate(() => ({
       marginTop: 1,
     },
   }),
+  clickableBoxStyle: Styles.platformStyles({
+    isElectron: {
+      flexShrink: 1,
+      height: 40,
+      width: 120,
+    },
+    isMobile: {
+      height: 48,
+    },
+  }),
   container: {
+    backgroundColor: Styles.globalColors.white,
     borderBottomColor: Styles.globalColors.black_10,
     borderBottomWidth: 1,
     borderStyle: 'solid',
-    flex: 1,
-    maxHeight: Styles.isMobile ? 48 : 40,
+    marginTop: Styles.globalMargins.tiny,
   },
   divider: {
     ...Styles.globalStyles.flexBoxRow,
     backgroundColor: Styles.globalColors.transparent,
     minHeight: 2,
   },
-  dividerSelected: {
-    ...Styles.globalStyles.flexBoxRow,
-    backgroundColor: Styles.globalColors.blue,
-    minHeight: 2,
-  },
+  dividerSelected: Styles.platformStyles({
+    common: {
+      ...Styles.globalStyles.flexBoxRow,
+      backgroundColor: Styles.globalColors.blue,
+      minHeight: 2,
+    },
+    isMobile: {minHeight: 3},
+  }),
   icon: {
     alignSelf: 'center',
   },
   iconSelected: {
     alignSelf: 'center',
     color: Styles.globalColors.black,
   },
+  minWidthFull: {minWidth: '100%'},
+  mobileDistribute: Styles.platformStyles({isMobile: {flex: 1}}),
+  mobileScroll: Styles.platformStyles({isMobile: {width: 120}}),
```

Comment:
does this need to be a fixed width?

Reply:
No, but it is in the designs https://zpl.io/291XDQY